cumulusci/tasks/salesforce/update_dependencies.py

```diff
@@ -245,51 +245,52 @@ def _install_dependencies(self):
 
     def _install_dependency(self, dependency):
         package_zip = None
-        if "zip_url" or "repo_name" in dependency:
-            zip_src = None
-            if "zip_url" in dependency:
-                self.logger.info(
-                    "Deploying unmanaged metadata from /{} of {}".format(
-                        dependency["subfolder"], dependency["zip_url"]
-                    )
-                )
-                zip_src = self._download_extract_zip(
-                    dependency["zip_url"], subfolder=dependency.get("subfolder")
-                )
-            elif "repo_name" in dependency:
-                self.logger.info(
-                    "Deploying unmanaged metadata from /{} of {}/{}".format(
-                        dependency["subfolder"],
-                        dependency["repo_owner"],
-                        dependency["repo_name"],
-                    )
-                )
-                gh_for_repo = self.project_config.get_github_api(
-                    dependency["repo_owner"], dependency["repo_name"]
+
+        zip_src = None
+        if "zip_url" in dependency:
+            self.logger.info(
+                "Deploying unmanaged metadata from /{} of {}".format(
+                    dependency["subfolder"], dependency["zip_url"]
```

Comment:
Line 257 hints that dependency["subfolder"] can be missing sometimes but this log message assumes that it is here.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/2149/commits/e79179e0976d706fd8544a1735b2739ea64e9314